Header header

float32 d_groundtruth
float32 d_raw_bb
float32 d_raw_alt
float32 bebop_alt
float32 beta
float32 sigma_1
float32 sigma_2
float32 cam_tilt
float32 target_height
float32 target_grounddist
uint16  bb_height

float32[6] v_img

float32 desired_depth
float32 desired_yaw_rad
